README.md

```diff
@@ -1,3 +1,41 @@
+## Marked wants you!
+
+In case you haven't noticed, and judging by attendance you haven't&hellip; (Movie. Major League. No? Anyway.)
+
+In case you haven't noticed, after years of [being mostly dead](https://github.com/markedjs/marked/issues/1106) (not to be confused with stable), Marked is trying to come back to life. We're about at the "You just wiggled your finger" stage ([The Princess Bride](https://youtu.be/yokQ0_8__ts). No? Come on!)
```

Comment:
I'm not the biggest fan of the conversation fluff.  It makes me take the project less seriously.